scss/tech.scss

```diff
@@ -31,9 +31,14 @@
     height: 100%;
     position: absolute;
     right: 10px;
+    transition: all 0.5s ease-in-out;
```

Comment:
It will be better to clarify which transitions you want to show instead of using `all`